contracts/core/EntryPoint.sol

```diff
@@ -55,15 +179,7 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuardT
     // Threshold below which no penalty would be charged
     uint256 private constant PENALTY_GAS_THRESHOLD = 40000;
 
-    /// @inheritdoc IERC165
-    function supportsInterface(bytes4 interfaceId) public view virtual override returns (bool) {
-        // note: solidity "type(IEntryPoint).interfaceId" is without inherited methods but we want to check everything
-        return interfaceId == (type(IEntryPoint).interfaceId ^ type(IStakeManager).interfaceId ^ type(INonceManager).interfaceId) ||
-        interfaceId == type(IEntryPoint).interfaceId ||
-        interfaceId == type(IStakeManager).interfaceId ||
-        interfaceId == type(INonceManager).interfaceId ||
-            super.supportsInterface(interfaceId);
-    }
+    SenderCreator private immutable _senderCreator = new SenderCreator();
```

Comment:
Please let's keep ALL constants and storage variables before ANY  function code?

Reply:
the reason of putting public/external first is to keep all external ABI first.
nothing about the private constants is external. I don't really see a reason to put them first, as they mean nothing outside their context.